src/main/java/be/seeseemelk/mockbukkit/inventory/meta/ItemMetaMock.java

```diff
@@ -315,71 +331,63 @@ else if (hasDamage())
 	@Override
 	public Set<Material> getCanDestroy()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		throw new UnimplementedOperationException("Deprecated method");
 	}
 
 	@Override
 	public void setCanDestroy(Set<Material> canDestroy)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		throw new UnimplementedOperationException("Deprecated method");
 	}
 
 	@Override
 	public Set<Material> getCanPlaceOn()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		throw new UnimplementedOperationException("Deprecated method");
 	}
 
 	@Override
 	public void setCanPlaceOn(Set<Material> canPlaceOn)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		throw new UnimplementedOperationException("Deprecated method");
 	}
```

Comment:
When were these deprecated? If it was after 1.8, or if the replacement methods weren't available by last update of the same major version they should be implemented or stay stubs. We only fail when calling methods deprecated before 1.8, or that had replacements by the end of the major version. For example, if they were added in 1.13 and were deprecated by 1.13.2, it's ok to fail the original.